src/main/kotlin/com/swisscom/health/des/cdr/clientvm/msal4j/LocalhostHttpClient.kt

```diff
@@ -0,0 +1,134 @@
+package com.swisscom.health.des.cdr.clientvm.msal4j
+
+import com.microsoft.aad.msal4j.HttpMethod
+import com.microsoft.aad.msal4j.HttpRequest
+import com.microsoft.aad.msal4j.HttpResponse
+import com.microsoft.aad.msal4j.IHttpClient
+import com.microsoft.aad.msal4j.IHttpResponse
+import java.io.DataOutputStream
+import java.io.IOException
+import java.io.InputStream
+import java.net.HttpURLConnection
+import java.net.URL
+import java.nio.charset.StandardCharsets
+import java.security.cert.X509Certificate
+import java.util.Scanner
+import javax.net.ssl.HttpsURLConnection
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+class LocalhostHttpClient : IHttpClient {
```

Comment:
Make it a Spring `@Component` and limit it to `@Profile("dev | test")` like the bean that is using the instance?